packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -187,7 +188,8 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
         );
     }
 
-    private renderTag = (tag: React.ReactNode, index: number) => {
+    private maybeRenderTag = (tag: React.ReactNode, index: number) => {
+        if (!tag) { return null; }
```

Comment:
well it's cool to go with React's semantics, but if you want to make the code more explicit you could go with `tag === false || tag == null`.